controllers/front/PageNotFoundController.php

```diff
@@ -60,4 +60,10 @@ public function getTemplateVarPage()
 
         return $page;
     }
+
+    public function displayAjax()
+    {
+        header('Content-Type: application/json');
```

Comment:
```suggestion
        header('Content-Type: application/json');
        header('HTTP/1.1 404 Not Found');
        header('Status: 404 Not Found');
```

Reply:
@PierreRambaud These headers have been set before, in [`PageNotFoundController::initContent()`](https://github.com/PrestaShop/PrestaShop/blob/develop/controllers/front/PageNotFoundController.php#L37) 😉